src/com/lftechnology/java/training/alina/basic/EvenFibonacci.java

```diff
@@ -0,0 +1,29 @@
+package com.lftechnology.java.training.alina.basic;
+
+import java.util.logging.Logger;
+
+/**
+ * Each new term in the Fibonacci sequence is generated by adding the previous
+ * two terms. By starting with 1 and 2, the first 10 terms will be: 1, 2, 3, 5,
+ * 8, 13, 21, 34, 55, 89, ... By considering the terms in the Fibonacci sequence
+ * whose values do not exceed four million. Find the sum of the even-valued
+ * terms.
+ * 
+ * @author Alina Shakya<alinashakya@lftechnology.com>
+ *
+ */
+public class EvenFibonacci {
+
+	private static final Logger LOGGER = Logger.getLogger(EvenFibonacci.class.getName());
+
+	public static void main(String[] args) {
+		try {
+			SumOfEvenFibonacci fibonacci = new SumOfEvenFibonacci(0, 1);
+			LOGGER.info("The sum of the even-valued fibonacci terms : "
```

Comment:
using parameterized logger is better when we are dealing with variable.
Have a look at:http://stackoverflow.com/questions/15880629/java-logging-with-multiple-parameters